crates/hyperswitch_connectors/src/connectors/fiservemea/transformers.rs

```diff
@@ -56,66 +92,261 @@ impl TryFrom<&FiservemeaRouterData<&PaymentsAuthorizeRouterData>> for Fiservemea
     ) -> Result<Self, Self::Error> {
         match item.router_data.request.payment_method_data.clone() {
             PaymentMethodData::Card(req_card) => {
-                let card = FiservemeaCard {
+                let card = FiservemeaPaymentCard {
                     number: req_card.card_number,
-                    expiry_month: req_card.card_exp_month,
-                    expiry_year: req_card.card_exp_year,
-                    cvc: req_card.card_cvc,
-                    complete: item.router_data.request.is_auto_capture()?,
+                    expiry_date: FiservemeaExpiryDate {
+                        month: req_card.card_exp_month,
+                        year: req_card.card_exp_year,
```

Comment:
Does it accepts both 2 digit and 4 digit year?

Reply:
It only accepts 2 digit expiry year and this bug is fixed.